x-pack/plugins/security/public/account_management/user_profile/user_profile.tsx

```diff
@@ -197,51 +220,41 @@ function UserSettingsEditor({
         }
         fullWidth
       >
-        <EuiButtonGroup
-          legend={i18n.translate(
+        <EuiKeyPadMenu
```

Comment:
Super Nit: I feel like I've looked at the other way (button group) too many times, but the row doesn't seem long enough. The text stating it's disabled because the config override has been set runs past the end of the keypad. I don't know how/if it should be changed 🤷‍♂️ Any ideas? if not, that is fine too!